script/publish_release.py

```diff
@@ -20,23 +20,23 @@ def main():
   repo = GitHub(get_env_var('GITHUB_TOKEN')).repos(BRAVE_REPO)
 
   release = get_draft(repo, get_brave_version())
-  commit_tag = get_commit_tag(get_brave_version())
+  commit_tag = get_commit_tag(get_raw_version())
 
   print("[INFO] Releasing {}".format(release['tag_name']))
   publish_release(repo, release['id'], get_tag(), commit_tag)
 
 def get_commit_tag(version):
-  parts = get_brave_version().split('.', 3)
+  parts = get_raw_version().split('.', 3)
```

Comment:
This part seems wrong if using the raw version, because it could return for example 0.50.7+67.0.3396.103 so it would have 6 parts, but later code checks for 3 parts.

Reply:
I don't think using the chrome version in the version string is going to work in a few places, i.e. the RPM version string is not going to support that. I chatted with @bsclifton this afternoon and he didn't think we want the chromium version in the package name at all. I'll ping you in the AM @bbondy to discuss further.